### PR TITLE
Add IDs to graph access logs

### PR DESCRIPTION
**Pull request by kobergj**
Fixes https://github.com/owncloud/ocis/issues/6576

### CODE REVIEW

File: services/graph/pkg/service/v0/educationclasses.go [hunk 1]
```diff
@@ -362,9 +362,9 @@ func (g Graph) PostEducationClassMember(w http.ResponseWriter, r *http.Request)
 // DeleteEducationClassMember implements the Service interface.
 func (g Graph) DeleteEducationClassMember(w http.ResponseWriter, r *http.Request) {
 	logger := g.logger.SubloggerWithRequestID(r.Context())
-	logger.Info().Msg("calling delete class member")
 
 	classID := chi.URLParam(r, "classID")
+	logger.Info().Str("classID", classID).Msg("calling delete class member")
```
Comment (micbar) [inline services/graph/pkg/service/v0/educationclasses.go:367]: we should also log the deleted member

**Comment by micbar** [comment]
I made some comments. I think these should apply to the other occurrances too. We should log what has changed.

File: services/graph/pkg/service/v0/educationclasses.go [hunk 2]
```diff
@@ -294,9 +294,9 @@ func (g Graph) GetEducationClassMembers(w http.ResponseWriter, r *http.Request)
 // PostEducationClassMember implements the Service interface.
 func (g Graph) PostEducationClassMember(w http.ResponseWriter, r *http.Request) {
 	logger := g.logger.SubloggerWithRequestID(r.Context())
-	logger.Info().Msg("Calling post class member")
 
 	classID := chi.URLParam(r, "classID")
+	logger.Info().Str("classID", classID).Msg("Calling post class member")
```
[the remark this answers is not in the document]
Comment (kobergj) [reply services/graph/pkg/service/v0/educationclasses.go:299]: We can only do that after we umarshalled the request body. I would need to move the log. That means if something fails while unmarshalling the body we would not have a access log of the request.

File: services/graph/pkg/service/v0/educationclasses.go (hunk 1 above)
Comment (kobergj) [reply services/graph/pkg/service/v0/educationclasses.go]: Did also with the other `DELETE` endpoints

**Review by micbar — APPROVED** [review]
Let us merge and see if that is sufficient to get a speaking access log

**Pull request merged**